src/scantailor/core/zones.py

```
@@ -0,0 +1,285 @@
+    def remove_picture_zone(self, zone: Zone) -> bool:
+        """Remove a picture zone.
+
+        Args:
+            zone: The zone to remove.
+
+        Returns:
+            True if zone was found and removed.
+        """
+        try:
+            self._picture_zones.remove(zone)
+            return True
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)